### DISCUSSION TITLE
Error while doing pw.x calculation

### DISCUSSION THREAD

**Discussion by Ezio-droid** [aiidateam/aiida-core#5452] [Q&A] [answered]
Hi all,

I am following the tutorial given on following link 
`[https://aiida-tutorials.readthedocs.io/en/tutorial-qe-short/source/sections/qe.html#importing-a-structure-and-inspecting-it]
And while I was seeing the process report for the job I got following errors. Can someone suggest the reason for these errors?
 
verdi process report 116
*** 116: None
*** (empty scheduler output file)
*** (empty scheduler errors file)
*** 3 LOG MESSAGES:
+-> CRITICAL at 2022-03-16 00:32:15.000564+00:00
 | Traceback (most recent call last):
 |   File "/home/tanay/miniconda3/envs/aiida/lib/python3.9/site-packages/aiida_quantumespresso/parsers/parse_raw/pw.py", line 434, in parse_stdout
 |     parsed_data['wall_time_seconds'] = convert_qe_time_to_sec(time)
 |   File "/home/tanay/miniconda3/envs/aiida/lib/python3.9/site-packages/aiida_quantumespresso/parsers/parse_raw/base.py", line 137, in convert_qe_time_to_sec
 |     num_seconds = (float(seconds) + float(minutes) * 60. + float(hours) * 3600. + float(days) * 86400.)
 | ValueError: could not convert string to float: 'PWSCF        :      2.46'
 | 
 | During handling of the above exception, another exception occurred:
 | 
 | Traceback (most recent call last):
 |   File "/home/tanay/miniconda3/envs/aiida/lib/python3.9/site-packages/aiida_quantumespresso/parsers/pw.py", line 325, in parse_stdout
 |     parsed_data, logs = parse_stdout(stdout, parameters, parser_options, parsed_xml)
 |   File "/home/tanay/miniconda3/envs/aiida/lib/python3.9/site-packages/aiida_quantumespresso/parsers/parse_raw/pw.py", line 436, in parse_stdout
 |     raise QEOutputParsingError('Unable to convert wall_time in seconds.')
 | aiida_quantumespresso.parsers.QEOutputParsingError: Unable to convert wall_time in seconds.
+-> ERROR at 2022-03-16 00:32:15.006631+00:00
 | The parser raised an unexpected exception.
+-> WARNING at 2022-03-16 00:32:15.009005+00:00
 | output parser returned exit code<350>: The parser raised an unexpected exception.

Thanks in advance

**Comment by chrisjsewell** [comment]
Heya, I would maybe open this as an issue on https://github.com/aiidateam/aiida-quantumespresso

**Comment by mbercx** [comment, marked as answer]
Moved to https://github.com/aiidateam/aiida-quantumespresso/issues/797
